crates/edr_provider/src/requests/methods.rs

```diff
@@ -386,7 +386,9 @@ pub enum MethodInvocation {
     SetStorageAt(
         #[serde(deserialize_with = "crate::requests::serde::deserialize_address")] Address,
         #[serde(deserialize_with = "crate::requests::serde::deserialize_storage_key")] U256,
-        #[serde(deserialize_with = "crate::requests::serde::deserialize_storage_value")] U256,
+        #[serde(deserialize_with = "crate::requests::serde::deserialize_storage_value")]
```

Comment:
Are you aware of the `with` attribute that allows you to specify the deserialization and serialization functions in one go, using a `mod`?

Reply:
Sure, I can change it if you want

Reply:
Circling back to this: I'm fine with either option. There are no `mod`s yet in `edr_provider::requests::serde`, so for me this version is more idiomatic in this repo, but `mod` is shorter. Let me know if I should change it please @Wodann !

Reply:
My preference is the one I left in the comment above, yes, so if you could change it that would be great.

If I'm not mistaken, this is a result of your recent additions of the alternative (serialisation) methods and not a general pattern, as can be seen elsewhere in the repo (and the serde docs).

Reply:
Ok cheers done in [dc94cd3](https://github.com/NomicFoundation/hardhat/pull/4952/commits/dc94cd32eaec8d6f53b278065ff66a186d717dfd)